ddl/db_test.go

```diff
@@ -1376,6 +1383,7 @@ func TestAmendTxnSavepointWithDDL(t *testing.T) {
 	tk := testkit.NewTestKit(t, store)
 	tk2 := testkit.NewTestKit(t, store)
 	tk.MustExec("use test;")
+	tk.MustExec("set global tidb_enable_mdl=0")
```

Comment:
Does set tidb_enable_mdl to 0 means disable mdl? As I understand, in default, current mdl is off, right?

Reply:
It's enabled now for testing.